docs/components/radio.md

```diff
@@ -239,67 +239,54 @@ Any valid attribute will be added to the input. You can read more about `$attrs`
 </KRadio>
 ```
 
-:::tip Note:
-When utilizing the `tooltip` slot, the `info` `KIcon` will be shown by default. To utilize the the `help` icon instead, set the `label-attributes` `help` property to any non-empty string value.
-:::
+## Events
 
-<KRadio v-model="tooltipChecked2" :selected-value="true" :label-attributes="{ help: 'true' }">
-  My tooltip
-  <template #tooltip>Brings all the <code>devs</code> to the yard</template>
-</KRadio>
+KRadio emits two events with same data in payloads.
 
-```html
-<KRadio
-  v-model="checked"
-  :label-attributes="{ help: 'true' }"
-  :selected-value="true"
->
-  My tooltip
-  <template #tooltip>Brings all the <code>devs</code> to the yard</template>
-</KRadio>
-```
+### change
 
-## Events
+Fired on change, returns the checked status of the radio (`boolean`).
+
+### update:modelValue
+
+Fired on change, returns the checked status of the radio (`boolean`).
```

Comment:
At the top of the docs page in the `selectedValue` docs, we say:

> The value of the KRadio option that will be emitted by the `change` and `update:modelValue` events.

Here in the events section, we don't mention that prop so it's a little confusing as it seems like we're emitting different data. Can we change the wording here?